website/code_issues.qmd

```diff
@@ -357,6 +357,30 @@ Ideally, use `requireNamespace("pkg")` or `require("pkg")`, both return `FALSE`
 
 
 
+--------------
+
+
+# Setting `options(warn = -1)`
+
+## Problem
+
+You are setting `options(warn = -1)`.
+
+## Solution
+
+Omit the code line and rather use `suppressWarnings()` if really needed.
```

Comment:
Small re-wording suggestion

```suggestion
Consider using `suppressWarnings()` instead of `options(warn = -1)` if you absolutely need to suppress warnings.
```